api/v1/helpers/verbs/utils.js

```diff
@@ -237,20 +237,53 @@ function handleAssociations(reqObj, inst, props, method) {
   });
 } // handleAssociations
 
+function isMultipleAssociations(opts) {
+  const MULTIPLE_ASSOCIATIONS = 3;
+  return opts.attributes.length >= MULTIPLE_ASSOCIATIONS;
```

Comment:
This will also return true when multiple non-association fields are specified.
I don't think this will cause problems, it just means we're doing extra work when not necessary. Correct?

Reply:
Good question. I thought that was the way to define 'multiple associations'.

Is there a correct way to check that? Then it will be better when flagging the fields to exclude before sending back to the client.

Reply:
This checks for multiple fields, but not all fields are associations. You would need to also check if the field has a foreign key, like you are doing elsewhere.

Reply:
Can you approve that and merge to master? then I can address that within the part 3. @jgraff2